### PR TITLE
Adding JaxNode to list of meetups

### PR DESCRIPTION
**Pull request by davidfekke**
Adding JaxNode to the list of meetups. We are the Node.js user group here in Jacksonville, Fl.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: locale/en/get-involved/node-meetups.md [hunk 1]
```diff
@@ -386,6 +386,17 @@ REQUIREMENTS
 - Organizer name - Ryan Connelly
 - Organizer contact info -
 
+#### Florida
```
Comment (ZYSzys) [inline locale/en/get-involved/node-meetups.md:379]: For lexicographical order, this should be put above Georgia.

Comment (Maledong) [reply locale/en/get-involved/node-meetups.md]: Helped to fix

Comment (ZYSzys) [reply locale/en/get-involved/node-meetups.md:379]: Ooops, I mean this whole addition(including the meetup) should be put above `Georgia`.😅

Jacksonville is in Florida

screen shot 2019-03-01 at 9 03 00 pm

Comment (ghost) [reply locale/en/get-involved/node-meetups.md:379]: OK

**Review by ZYSzys — APPROVED** [review]

**Comment by ZYSzys** [comment]
@davidfekke @Maledong Thank you !👍

**Pull request merged**